### ISSUE TITLE
NTR: MRI: heart iron content increased (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8997]
New term request MRI: heart iron content increased (elthat_140318190046): Cardiac  UMLS: 4552187

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000663
- label: Elevated myocardial iron load
- parents: [Abnormal myocardium morphology (HP:0001637)](https://hpo.jax.org/app/browse/term/HP:0001637)
- pmid: PMID:28458697|PMID:16888797
- def: Increased amount of iron in myocardial tissue.
- comment: This feature can be assesed by magnetic resonance imaging (MRI). MRI as a reliable, valid, and robust method can provide indirect assessment of cardiac iron overload. Even, it has been advised to determination the degree of cardiac iron overload.8 The reciprocals of T2 and T2*, known as R2 and R2*, are directly proportional to iron. This feature can be seen as a consequence of transfusion treatment in thalassemia.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8997
- synonyms: Cardiac iron overload(NONE)|MRI: heart iron content increased(NONE)
